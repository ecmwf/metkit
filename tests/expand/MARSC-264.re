retrieve,
    class=od,
    type=fcmax,
    stream=sfmm,
    expver=0001,
    repres=gg,
    levtype=sfc,
    param=167,
    date=19911001,
    fcmonth=02/03/04,
    time=0000,
    number=0/1/2/3/4,
    domain=g,
    system=02,
    method=01,
    source="mod.cl.surface",
    target="data.reference"