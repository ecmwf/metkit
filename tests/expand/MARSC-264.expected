 RETRIEVE,
    CLASS      = OD,
    TYPE       = FCMAX,
    STREAM     = SFMM,
    EXPVER     = 0001,
    REPRES     = GG,
    LEVTYPE    = SFC,
    PARAM      = 167,
    DATE       = 19911001,
    FCMONTH    = 2/3/4,
    TIME       = 0000,
    NUMBER     = 0/1/2/3/4,
    DOMAIN     = G,
    SYSTEM     = 2,
    METHOD     = 1,
    SOURCE     = mod.cl.surface,
    TARGET     = data.reference,
    DATABASE   = file